src/appengine/private/components/upload-testcase/upload-form.html

```diff
@@ -276,6 +279,13 @@
             window.query.get(), this.paramKeys);
       }
 
+      shouldShowFiletypePicker(filePath) {
+        if (!filePath) {
+          return false;
+        }
+        return !filePath.includes('.html') && !filePath.includes('.js') && !filePath.includes('.pdf');
```

Comment:
Sorry why is this part needed?

Reply:
to show the message only when the file type is an archive! felt like it didnt need to be shown otherwise